### DISCUSSION TITLE
Question About Exporting, Modifying, and Re-Importing Applications in Budibase

### DISCUSSION THREAD

**Discussion by guibrasa** [Budibase/budibase#15016] [Help] [answered]
Hi everyone,

First of all, thank you for the amazing application! I'm just starting my journey with Budibase, and I'm really enjoying it so far.

I had an idea about one of its features. I noticed that it's possible to export an application and import it again later. Regarding this, I saw that the export file is essentially code. So, I tried modifying the code to re-import the application with some additional features. However, after making changes to the code, I was unable to import the application back into Budibase.

My question is: Is it possible to create or modify an export code and then import it into Budibase? I **attempted to use AI to generate code and simplify the app creation process**.

Thank you for your attention and support!

**Comment by mike12345567** [comment, marked as answer]
Hi @guibrasa - today this is something that isn't possible - in theory it might be possible to make *small* changes to the export without breaking things, but the format is a raw DB export - it is not easily editable and requires very in-depth knowledge of the format of the components that make up a Budibase app.

It is possible if there was interest in it that we might someday offer a git export functionality that would export the app in a more human readable/editable format, but there is nothing like this on the roadmap today.
